src/main/java/seedu/task/model/task/Complete.java

```diff
@@ -0,0 +1,34 @@
+package seedu.task.model.task;
+
+import seedu.task.commons.exceptions.IllegalValueException;
+
+public class Complete {
+
+    public boolean completion = false;
+
+    /**
+    * Validates given description.
+    *
+    * @throws IllegalValueException if given description string is invalid.
+    */
+    public Complete(boolean completion) {
+        this.completion = completion;
+    }
+
+    public void setComplete() {
+        this.completion = true;
+    }
+
+    public void setNotComplete() {
```

Comment:
nit: is there a reason why we didn't combine the setComplete and setNotComplete into one method?

public void setComplete(boolean isComplete) {
       this.completion = isComplete;
}